src/nodes/ImageView.vue

```diff
@@ -124,15 +142,15 @@ export default {
 	name: 'ImageView',
```

Comment:
What do you think about `MediaView` or `AttachmentView`?

Reply:
Same here, the ImageView node still renders the markdown image elements.